openlibrary/templates/books/edit/edition.html

```diff
@@ -427,7 +390,10 @@
                     $_("WARNING: Any edits made to the work from this page will be ignored.")
                 </span>
             </div>
-            <div id="works">
+            <div class="works_multi_input_autocomplete"
+                 data-isPrivilegedUser="$cond(is_privileged_user, 'true', 'false')"
+                 data-newWorkText="$_('-- Move to a new work')"
```

Comment:
I think we should put these both in`data-config`. We're using that pattern in other templates so it makes it easier to find where we are doing this in future if they are used consistently.